util/Cargo.toml

```diff
@@ -35,20 +35,20 @@ sha3 = "0.8"
 # grin_store = { git = "https://github.com/mimblewimble/grin", tag = "v4.0.0-rc.1" }
 
 # For bleeding edge
-grin_core = { git = "https://github.com/mimblewimble/grin", branch = "master" }
```

Comment:
Careful we don't commit these changes and revert this file (and lock file) before merging this.

Reply:
Yep that what I told you I was about to remove on Keybase